tools/autograd/derivatives.yaml

```diff
@@ -650,6 +650,9 @@
 
 - name: zeros  # fallthrough
 
+- name: _standard_gamma(Tensor self, Generator generator)
+  self: grad * output._standard_gamma_grad(self)
```

Comment:
The syntax and naming here is different from the other derivative definitions. (output becomes self and self becomes alpha). It doesn't look like _standard_gamma_grad is used outside of the derivative definition. Can we re-order and re-name the arguments? e.g.

```
_standard_gamma_grad(self, output).mul_(grad)
```

or fuse the gradient multiplication:

```
_standard_gamma_grad(grad, self, output)
```

Reply:
I reordered the parameters; I didn't fuse the grad multiplication because I don't know if this subcomputation is going to be reused in another derivative.